src/components/OcButton.vue

```diff
@@ -1,13 +1,12 @@
 <template>
   <component
     :is="type"
-    :href="href"
-    :to="to"
+    :v-bind="properties"
```

Comment:
`v-` directives don't have the leading colon. Please change to `v-bind` (instead of `:v-bind`)